src/api-routes/persons/{osuId}/images.js

```diff
@@ -15,14 +15,20 @@ const { maximum: maxWidth } = openapi.components.parameters.imageWidth.schema;
 const get = async (req, res) => {
   try {
     const { osuId } = req.params;
-    const { width } = parseQuery(req.query);
+    const { width } = req.query;
     const image = await getImageById(osuId);
 
+    // return default image if no image is returned from data source
     if (!image) {
-      return errorBuilder(res, 404, 'No image with the specified OSU ID was not found.');
+      return fs.readFile('src/resources/defaultImage.jpg', (err, data) => {
```

Comment:
Looks like `width` parameter doesn't work on default image. Can you take a look?

Reply:
It doesn't work by design but I can change that.